### ISSUE TITLE
expected italics

### ISSUE THREAD

**Issue by mrutgers** [robconery/troys-book#75]
pdf version 001 page 444, reading with Edge on Windows 11

Minor quibble: On this page the first quoted snippet is in italics, so I would expect the second snippet to be in italics too

**Comment by troyhunt** [comment]
I agree with this one @robconery, it's in the epilogue for how I decide what I endorse.
